Dockerfile.plugin

```diff
@@ -14,11 +14,11 @@ FROM alpine:3.21 AS runtime-stage
 WORKDIR /
 
 ARG port=55555
-COPY --from=build-stage /camino-messenger-bot/plugin /plugin
+COPY --from=build-stage /camino-messenger-bot/pp-mock /pp-mock
```

Comment:
This line is not correct, it copies the "pp-mock" folder into the container. The old build command above was building a "plugin" binary into the current directory but the new build script is building it into the build directory.